prov/gni/src/gnix_wait.c

```diff
@@ -195,17 +347,13 @@ static int gnix_init_wait_obj(struct gnix_fid_wait *wait, enum fi_wait_obj type)
 	wait->type = type;
 
 	switch (type) {
-	case FI_WAIT_FD:
```

Comment:
The code was defined previously as boiler plate for use in the future, but as such we cannot support FD or COND based on the current implementation so I removed the extraneous code for now.